test/common_nn.py

```diff
@@ -295,7 +295,7 @@ def bceloss_no_reduce_test():
             lambda i: F.binary_cross_entropy(i, t.type_as(i), reduction='none')),
         input_fn=lambda: torch.rand(15, 10).clamp_(2.8e-2, 1 - 2.8e-2),
         reference_fn=lambda i, *_: -(t * i.log() + (1 - t) * (1 - i).log()),
-        check_gradgrad=False,
+        check_gradgrad=True,
```

Comment:
nit: `check_gradgrad` is the default, so you can just delete this line and the ones below.